scripts/sitemap.js

```diff
@@ -35,7 +35,6 @@ const urls = [
     ['regulation',                         'monthly', 0.80, 'id'],
     ['responsible-trading',                'monthly', 0.80],
     ['terms-and-conditions',               'monthly', 0.80],
-    // ['user/browser-support',               'monthly', 0.80],
```

Comment:
if we're deleting it then please remove it rather than commenting. if we might return it, then we should comment the line in pages.js as well